### DISCUSSION TITLE
Protocols handled by SavvyCAN?

### DISCUSSION THREAD

**Discussion by CKRS2** [collin80/SavvyCAN#355] [Q&A]
As a great fan of SavvyCAN, i'm into a new project.

From scanning through the SavvyCAN software followed by the ESP32RET/M2RET software I conclude that all the protocols are handled by SavvyCAN and not by the vehicle interface itself. Am I correct on this one?

If this is true, this basically means that the vehicle interface up through the PC are actually an extension of the physical bus?!

Looking forward to your reply!

**Comment by collin80** [comment]
Yes, SavvyCAN merely grabs the raw CAN frames from the vehicle interface. From there they can be interpreted in a variety of ways. Yes, it is correct to look at it as an extension of the physical bus.
